src/routes/admin-api/route.ts

```
@@ -1652,3 +1653,73 @@ adminApiRoutes.post("/accounts/:id/reauth", async (c) => {
+  const p = url.searchParams
+
+  const from = p.get("from") || undefined
+  const to = p.get("to") || undefined
+  const accountId = p.get("account_id") || undefined
+  const granularity = p.get("granularity") === "hour" ? "hour" : "day"
+
+  const now = new Date()
+  const todayStr = `${now.getFullYear()}-${String(now.getMonth() + 1).padStart(2, "0")}-${String(now.getDate()).padStart(2, "0")}`
+
+  const resolvedFrom =
+    from ||
```

ESLint

[error] Delete `·||`

(prettier/prettier)